src/Altinn.Notifications/Validators/Rules/RecipientRules.cs

```diff
@@ -234,7 +234,7 @@ internal static bool IsValidEmail(string? email)
             return false;
         }
 
-        string emailRegexPattern = @"((&quot;[^&quot;]+&quot;)|(([a-zA-Z0-9!#$%&amp;'*+\-=?\^_`{|}~])+(\.([a-zA-Z0-9!#$%&amp;'*+\-=?\^_`{|}~])+)*))@((((([a-zA-Z0-9������]([a-zA-Z0-9\-������]{0,61})[a-zA-Z0-9������]\.)|[a-zA-Z0-9������]\.){1,9})([a-zA-Z]{2,14}))|((\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})))";
+        string emailRegexPattern = @"((&quot;[^&quot;]+&quot;)|(([a-zA-Z0-9!#$%&amp;'*+\-=?\^_`{|}~])+(\.([a-zA-Z0-9!#$%&amp;'*+\-=?\^_`{|}~])+)*))@((((([a-zA-Z0-9������]([a-zA-Z0-9\-������]{0,61})[a-zA-Z0-9������]\.)|[a-zA-Z0-9������]\.){1,9})([a-zA-Z]{2,14}))|((\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})))";
```

Comment:
Something weird going on with the norwegian letters.

Reply:
Well spotted! Fixed in a new commit

Reply:
Still not right

Reply:
I've found the issue - this file is saved with ISO-8859-1 encoding ("Western"). 
If you [open RecipientRules.cs on the main branch for edit](https://github.com/Altinn/altinn-notifications/edit/main/src/Altinn.Notifications/Validators/Rules/RecipientRules.cs), you'll see the "warning" message as shown in the attached image.

![bilde](https://github.com/user-attachments/assets/87a3a836-274f-4a42-b6b7-31faa2dc569e)

This can also be seen by inspecting the file on `main` in VS Code. Comparing encodings ([link to how to change encoding](https://stackoverflow.com/questions/30082741/change-the-encoding-of-a-file-in-visual-studio-code)), the letters æøå etc. are gibberish in UTF-8, but fine in ISO-8859-1.

My hypothesis is the following: this PR successfully saves RecipientRules.cs with UTF-8 encoding. It looks gibberish here because GitHub is showing the file changes _**using the previous ISO-8859-1 encoding**_.

You can verify this locally in VS Code by checking out this branch and inspecting RecipientRules.cs

Reply:
Might be isolated to the diff tool. When I open the file the Norwegian letters are displayed correctly. Could you expand the tests with one test for an email that includes a Norwegian letter?

https://github.com/Altinn/altinn-notifications/blob/860ceac07a12ef3b04e9e431040acc945501e1e8/test/Altinn.Notifications.Tests/Notifications/TestingValidators/EmailNotificationOrderRequestValidatorTests.cs#L227